core/modules/system/system.module

```diff
@@ -1842,14 +1842,24 @@ function system_library_info() {
       array('system', 'ui'),
     ),
   );
+  $libraries['jquery.color'] = array(
+    'title' => 'jQuery Color',
+    'website' => 'https://github.com/jquery/jquery-color',
+    'version' => '2.2.0',
+    'js' => array(
+      'core/misc/ui/vendor/jquery-color/jquery.color.js'  => array('group' => JS_LIBRARY, 'weight' => -9),
```

Comment:
Could you place this directly in `/misc/jquery.color.js` to match other jQuery libraries? Then we'll exclude the entire `vendor` directory from the `/ui/` directory.